server/controllers/finance/reports/collection_capacity/index.js

```diff
@@ -0,0 +1,102 @@
+/**
+ * @overview ./finance/reports/collection_capacity/
+ */
+
+const _ = require('lodash');
+const moment = require('moment');
+
+const ReportManager = require('../../../../lib/ReportManager');
+const db = require('../../../../lib/db');
+
+module.exports.report = report;
+
+// path to the template to render
+const TEMPLATE = './server/controllers/finance/reports/collection_capacity/report.handlebars';
+
+const DEFAULT_OPTIONS = {
+  orientation : 'portrait',
+  footerRight : '[page] / [toPage]',
```

Comment:
Both the `footerRight` and the `footerFontSize` properties are no longer necessary.  These were from wkhtmltopdf and we are using puppeteer now.